src/ImageRunModal.scss

```diff
@@ -65,6 +65,15 @@
     }
 }
 
+.pf-m-3-col-on-md .pf-c-select__toggle-typeahead {
+    flex-basis: 5rem;
+    min-width: 5rem;
+}
```

Comment:
FWIW: This will affect everything on the page, even outside of the modal.

Reply:
Right, should I give it special class for better targeting only of this special case?

Reply:
I already broke it :smile: Because I changed the `3` to `4`

Reply:
We also shouldn't key CSS on highly specific, fragile identifiers, like `pf-m-3-col-on-md`. :wink: